packages/gigamon/data_stream/ami/fields/ml-dga.yml

```
@@ -0,0 +1,12 @@
+- name: ml_is_dga
+  type: group
+  fields:
+    # dga fields
```

I think this comment could be more explicit.
```suggestion
    # Domain Generation Algorithm fields
```
Also, can we have description for the fields below?

---

Changes made as per the suggestion